routes/recipe.js

```diff
@@ -43,6 +43,24 @@ const getRecipeByID = (request, response) => {
         });
 };
 
+//GET ALL RECIPES FOR A USER BY USER ID
+const getRecipesByUserID = (request, response) => {
+    const user_id = request.params.id;
```

Comment:
With the change in the route this would just become `const { user_id, } = request.params;`

Reply:
I will also update this.